cola/icons.py

```diff
@@ -31,9 +32,14 @@
     '.cxx':     'file-code.svg',
 }
 
+cfg = gitcfg.current()
+style = cfg.icon_style()
 
 def install():
-    icon_dir = resources.icon_dir()
+    if not style or style == "light":
+        icon_dir = resources.icon_dir()
+    else:
+        icon_dir = resources.icon_dir(style)
```

Comment:
tiny tweak: for keyword arguments, prefer doing `icon_dir(style=style)` at the callsite.